### PR TITLE
Blog-App Controllers specs

### PR DESCRIPTION
**Pull request by Kossi-stack**
**In this pull request these are the following requirements**

- Add RSpec as a gem to your project.
- Create a Request spec file for all your controllers.
- Make sure that for each action you check:

        If response status was correct.

        If a correct template was rendered.

        If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    it 'renders correct template' do
+      expect(response).to_not render_template('index')
+    end
+
+    it 'check for placeholder text' do
+      expect(response.body).to include('Post 10 by User')
+    end
+  end
+end
```
Comment (AshakaE) [inline spec/requests/posts_spec.rb]: - [ ] Please add the describe block of `/show` for the post controller

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @Kossi-stack  🙋‍♀️, 

Good job so far! 👌

Your:
   - Git flow ✔️ 
   - Pull request 👍 
   - README 👌 
   
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/posts_spec.rb (hunk 1 above)
Comment (Kossi-stack) [reply spec/requests/posts_spec.rb]: Thanks just fix it

**Review by AshakaE — APPROVED** [review]
### Hi @Kossi-stack   🙋‍♀️, 

# STATUS APPROVED ✔
   
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**